### ISSUE TITLE
Sketches spawn off screen on Windows 10.

### ISSUE THREAD

**Issue by Bracktus** [soegaard/sketching#21]
After pressing run in DrRacket with the sketch spawns off screen.

In order to view the sketch (at least in windows 10) you need to Shift+Right-Click the window on the taskbar to open a menu and then select maximise.

**Comment by soegaard** [comment]
Hi @Bracktus 

I have pushed a fix to this issue. 

Update the `sketching` package to get it.

The sketch was set to spawn at (screenwidth,0).

On macOS the sketch would spawn in the upper, right hand corner of the screen - and be visible.
On Windows the sketch would spawn just right of the visible area.

I have solved the issue, by not specifying a particular coordinate set - which means
the placement is the OS default. On macOS this is the upper, left corner of the screen.

Thanks for the bug report.
